wiki/osu!api/pt-br.md

```diff
@@ -0,0 +1,15 @@
+# osu!api
+
+osu! tem uma [API](https://pt.wikipedia.org/wiki/Interface_de_programa%C3%A7%C3%A3o_de_aplica%C3%A7%C3%B5es) pública que podem ser usados por serviços terceirizados para expandir a plataforma osu!. A API costumava ser mais privada, usada apenas por aqueles que a solicitavam, mas agora qualquer um pode usar a API desde que tenha uma conta osu!.
+
+Você pode entcontrar a documentação da API na [Wiki da osu-api](https://github.com/ppy/osu-api/wiki). A versão atual da API usa chaves privadas para autenticar requests. Você pode aplicar para uma chave da API na [página da aplicação](https://osu.ppy.sh/p/api) no site do osu!. Note que a documentação na wiki pode não ser completa ou extensiva, e a API não está em desenvolvimento ativo no momento.
```

Comment:
```suggestion
Você pode entcontrar a documentação da API na [Wiki da osu-api](https://github.com/ppy/osu-api/wiki). A versão atual da API utiliza de chaves privadas para autenticar solicitações. Você pode se aplicar para uma chave da API na [página de aplicação](https://osu.ppy.sh/p/api) no site do osu!. Note que a documentação na wiki do osu!api pode não estar completa ou extensiva, e a API em si não está atualmente em desenvolvimento.
```
Ease of comprehension. Similar writing suggested.
`solicitações` is a valid term when developing with APIs in PT-BR so it is okay to use it instead of `requests`